### ISSUE TITLE
Support API to Update Voting History

### ISSUE THREAD

**Issue by nElder03** [ChapmanCPSC/SE-498_01#170]
## Description
```
As a ,
I can ,
so that .
```

## Acceptance Criteria
1.

**Comment by lancefluger** [comment]
@lavin105 we have this on the frontside correct? We just need to make the api call? Do we have the backend service ready? 

@andrewdieken

**Comment by lancefluger** [comment]
@andrewdieken  - Let me know if have capacity for this story

**Comment by lavin105** [comment]
Yes i will have to modify 3 lines and the call is already made the score
will just need to he added for every voter and of they have no score set to
0
